lib/key/utils.js

```diff
@@ -130,8 +188,7 @@ export async function getMatchingKey(signature, keys) {
 }
 
 export async function cloneKey(inputKey) {
-    const [key] = await getKeys(inputKey.toPacketlist().write());
-    return key;
+    return getKey(inputKey.toPacketList().write());
```

Comment:
This is not needed anymore (?), thanks to the cloning performed by `openpgp.encryptKey`